source/_integrations/sleepiq.markdown

```diff
@@ -13,21 +13,26 @@ ha_platforms:
   - sensor
 ---
 
-The SleepIQ implementation lets you view sensor data from [SleepIQ by SleepNumber](https://www.sleepnumber.com/sleepiq-sleep-tracker). In particular, it lets you see the occupancy and current SleepNumber (ie current firmness) of each side of a SleepNumber bed.
+The SleepIQ integration lets you view sensor data from [SleepIQ by SleepNumber](https://www.sleepnumber.com/sleepiq-sleep-tracker). In particular, it lets you see the occupancy and current SleepNumber (ie current firmness) of each side of a SleepNumber bed.
 
-## Setup
-
-You will need an account on [SleepIQ](https://sleepiq.sleepnumber.com/) to use this component.
+You will need an account on [SleepIQ](https://sleepiq.sleepnumber.com/) to use this integration.
 
 ## Configuration
 
-To set it up, add the following information to your `configuration.yaml` file:
+1. In the **Configuration** > **Devices & Services** menu, click **+** and then select "SleepIQ" from the pop-up menu.
+2. In the pop-up box, enter your username and password for your SleepIQ account. Optionally, choose a custom scan interval (the default is to check for updates every 60 seconds).
+3. Click **Submit**. If the account information is valid, a configuration entry will be created and your SleepNumber sensors will be available in Home Assistant.
```

Comment:
Below code will add a big button, making it possible for the user to open the config flow on their Home Assistant instance directly.

```
{% include integrations/config_flow.md %}
```

Reply:
Done, swapped out the manual steps for the button